### DISCUSSION TITLE
How do I use turn signals manually

### DISCUSSION THREAD

**Discussion by CrossLoxx** [YimMenu/YimMenu#1600] [Q&A]
I want to use turn signals because it makes gta a bit more immersive

**Comment by alyxme** [comment]
already in the menu: Go to Vehicle then enable checkbox Turnsignals. Press J for left turn, press L for right turn and K for both (hazzard blinkers)

**Reply by LintStalker** [reply]
OMG, that's insane!!!  This is such an awesome menu!
